src/server/storage/storageAPI.js

```diff
@@ -182,6 +194,39 @@ const setup = (app: Router, gunPublic: StorageAPI, storage: StorageAPI) => {
     })
   )
 
+  /**
+   * @api {post} /user/start user starts registration and we have his email
+   * @apiName Add
+   * @apiGroup Storage
+   *
+   * @apiParam {Object} user
+   *
+   * @apiSuccess {Number} ok
+   * @ignore
+   */
+  app.post(
+    '/user/start',
+    onlyInEnv('production', 'staging', 'test'),
+    passport.authenticate('jwt', { session: false }),
+    wrapAsync(async (req, res) => {
+      const { user } = req.body
+      const { log: logger, user: existingUser } = req
+      const { __utmzz: utmString = '' } = req.cookies
+
+      if (!user.email || existingUser.createdDate || existingUser.mauticId) return res.json({ ok: 0 })
+
+      await addUserSteps
+        .updateMauticRecord(user, utmString, logger)
+        .then(r => logger.debug('updateMauticRecord success'))
+        .catch(e => {
+          logger.error('updateMauticRecord failed', e.message, e, { user })
+          throw new Error('Failed adding user to mautic')
```

Comment:
will this be caught in wrapAsync ?
otherwise - need to return { ok: 0, error: 'Failed adding user to mautic' } explicitly

Reply:
yes it will